src/journal-remote/microhttpd-util.c

```diff
@@ -103,7 +103,10 @@ int mhd_respondf(struct MHD_Connection *connection,
         errno = -error;
         fmt = strjoina(format, "\n");
```

Comment:
The commit message is a copy&paste error, as in this case we are actually appending a known value (newline). But this is not fundamentally misleading, so no need (from my side) to repush just for this.